commit-msg-hook.js

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env node
 
 // `commitplease-original`
-require('commitplease')(process.argv[ 2 ])
+require('commitplease')()
```

Comment:
I think this should now be just `require('commitplease')`, since that module executes itself.